amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -18,41 +18,20 @@ export type TableDataAPI= { tableData: TableData; } & MessageAPI;
 
 /** HELPERS **/
 import {
-  getTableQueryParams, getTableDataFromResponseData, getTableOwnersFromResponseData, getTableTagsFromResponseData,
+  getTableQueryParams, getTableDataFromResponseData, getTableOwnersFromResponseData,
   createOwnerUpdatePayload, createOwnerNotificationData, shouldSendNotification
 } from './helpers';
 
-export function getTableTags(tableKey: string) {
-  const tableParams = getTableQueryParams(tableKey);
-  return axios.get(`${API_PATH}/table?${tableParams}`)
-  .then((response: AxiosResponse<TableDataAPI>) => {
-    return getTableTagsFromResponseData(response.data);
-  });
-}
-
-/* TODO: Typing this method generates redux-saga related type errors that needs more dedicated debugging */
-export function updateTableTags(tagArray, tableKey: string) {
-  const updatePayloads = tagArray.map((tagObject) => {
-    return {
-      method: tagObject.methodName,
-      url: `${API_PATH}/update_table_tags`,
-      data: {
-        key: tableKey,
-        tag: tagObject.tagName,
-      },
-    }
-  });
-  return updatePayloads.map(payload => { axios(payload) });
-}
 
+// TODO - Revisit the necessity of filtering out tags from main metadata
 export function getTableData(tableKey: string, index?: string, source?: string ) {
   const queryParams = getTableQueryParams(tableKey, index, source);
   return axios.get(`${API_PATH}/table?${queryParams}`)
   .then((response: AxiosResponse<TableDataAPI>) => {
     return {
       data: getTableDataFromResponseData(response.data),
       owners: getTableOwnersFromResponseData(response.data),
-      tags: getTableTagsFromResponseData(response.data),
+      tags: response.data.tableData.tags,
```

Comment:
Still needs alphabetical sorting.

Reply:
I think I can actually remove `tags` field here and add it back into the `data` which filters it out. Then in the Tags reducer and sagas we can read `response.data.tableData.tags` instead of `response.tags`. I'll add this as a follow up PR since there are a lot of places that need to be changed.

Reply:
See comment above: https://github.com/lyft/amundsenfrontendlibrary/pull/426#discussion_r413414960

Please consider the `globalState` if you make changes. It is recommended to set up application state such that each object is stored once in the globalState.